src/org/parosproxy/paros/extension/option/OptionsViewPanel.java

```diff
@@ -513,6 +531,22 @@ public void stateChanged(ChangeEvent e) {
 		return fontSize;
 	}
 	
+    private ZapNumberSpinner getHttpMessageFontSize() {
+		if (httpMessageFontSize == null) {
+			httpMessageFontSize = new ZapNumberSpinner(-1, 8, 100);
+			if (! FontUtils.canChangeSize()) {
+				httpMessageFontSize.setEnabled(false);
+			}
+			httpMessageFontSize.addChangeListener(new ChangeListener() {

```

Comment:
This has no effect, the method `setExampleFont` uses the other font/size. Probably better to add another example entry.